x-pack/solutions/security/plugins/security_solution/public/flyout/entity_details/generic_right/index.tsx

```diff
@@ -56,6 +59,9 @@ export interface GenericEntityPanelExpandableFlyoutProps extends FlyoutPanelProp
 }
 
 export const GenericEntityPanel = ({ entityDocId, scopeId }: GenericEntityPanelProps) => {
+  const { uiSettings } = useKibana().services;
+  const assetInventoryEnabled = uiSettings.get(ENABLE_ASSET_INVENTORY_SETTING, true);
```

Comment:
i might be missing context but is this check needed? isnt the whole page behind this feature flag? so if its not enabled the entire flyout wont even show?